docs/JavaScript/README.md

```diff
@@ -4,9 +4,6 @@
 
 -   [General Guidelines](#general-guidelines)
 -   [Style Guide](#style-guide)
-    -   [Naming Conventions](#naming-conventions)
```

Comment:
In this case, if we just use airbnb guide, snake_case isn't even allowed in their configuration. Still we need to override this rule for the time being and ask the team to write in camelCase from now own as it's an industry standard. Right?

Reply:
Thanks @shahzaib-deriv @ali-hosseini-deriv . My thoughts were to move slowly on this one and start by updating the docs here first, then we can gradually start moving over to the airbnb convention.